solutions/deploy-argocd-apps-pull/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+# Troubleshooting
+
+#### For ArgoCD components, check the following containers for logs:
+* argocd-pull-integration-* in the `open-cluster-management` namespace (only on the hub cluster)
+* argocd-applicationset-controller
```

Comment:
This is in the `argocd` namespace.

Reply:
![image](https://github.com/open-cluster-management-io/ocm/assets/15898988/0e2da002-8391-44a3-8f1a-795b8ded4066)